docs/index.html.handlebars

```diff
@@ -9,19 +9,22 @@
 <body>
     <input type="checkbox" id="open-nav">
     <header class="navbar navbar-fixed-top navbar-inverse container-fluid">
-        <div class="navbar-header">
-            <label class="open-nav" for="open-nav"></label>
-            <a class="navbar-brand" href="/">
-                <strong>Ramda</strong>
-                <span class="version">v{{version}}</span>
-            </a>
-        </div>
-        <div class="navbar-left">
-            <ul class="nav navbar-nav">
-                <li><a href="..">Home</a></li>
+        <div class="container-fluid">
+            <div class="navbar-header">
+                <label class="open-nav" for="open-nav"></label>
+                <a class="navbar-brand" href="#">
+                    <strong>Ramda</strong>
+                    <span class="version">v{{version}}</span>
+                </a>
+            </div>
+            <ul class="nav navbar-nav navbar-left">
+                <li><a href="../">Home</a></li>
                 <li class="active"><a href="#">Documentation</a></li>
                 <li><a href="/repl?v={{version}}">Try Ramda</a></li>
+            </ul>
+            <ul class="nav navbar-nav navbar-right">
                 <li><a href="https://github.com/ramda/ramda">GitHub</a></li>
+                <li><a href="https://gitter.im/ramda/ramda">Gitter</a></li>
```

Comment:
That would make it easier to spot beside "GitHub".

Would the use of "Gitter" elsewhere make it seem like "Discuss" goes somewhere else?

I'll also offer "Chat" as an alternative, to match the "Open chat" Gitter Sidecar link.
That said, I have no major preference between any of them.